src/main/java/mat/server/service/impl/MeasurePackageServiceImpl.java

```diff
@@ -351,6 +351,16 @@ public int saveAndReturnMaxEMeasureId(final Measure measure) {
         return measureDAO.saveandReturnMaxEMeasureId(measure);
     }
 
+    @Override
+    public int returnMaxEMeasureId() {
```

Comment:
`MaxEMeasureId` is misleading. If it were returning the max eMeasure ID, then it should return the same value on every call, the largest possible eMeasure ID. Suggest `generateEMeasureId()` or `getNextEMeasureId()`.

Reply:
Yes, I will work on renaming it.

Reply:
I made changes, Can you take a look at them?

Reply:
Looks good!